### ISSUE TITLE
Sawroom wallet doesn't support fractions

### ISSUE THREAD

**Issue by puria** [dyne/sawroom-storage-tp#1]
@andrea-dintino:

Intkey doesn't support fractions, only ints.

In order to manage fractions, we convert them in Restroom-mw, like:
1 Sawroom "intkey" coin = 0.000001 Sawroom coins

**Comment by andrea-dintino** [comment]
Yes, but we can divide by 100000 doing:  Sawroom "intkey" coin = 0.000001 Sawroom coins

**Comment by andrea-dintino** [comment]
Yes
